recipes/pygwb/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "pygwb" %}
+{% set version = "1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygwb-{{ version }}.tar.gz
+  sha256: 675da6be45411e39b0032913499de3189abcf2f3bc4e48ff3bd208f86d22fa1d
+
+build:
+  skip: true  # [win or py>=312]
```

It's preferable to have noarch: python recipes. We can merge if CI is failing for missing dependencies on Windows.
```suggestion
  noarch: python 
```

---

@xhochy, the package uses setuptools `scripts`, so I don't think it's appropriate for noarch just yet.

---

Ah, was missing this context!